---
header:
  image: test2.jpg
title: "Complex rearrangements and oncogene amplifications revealed by long-read DNA and RNA sequencing of a breast cancer cell line."
collection: publications
permalink: /publication/2018-06-27-complex_rearrangements_and_oncogene_amplifications-18
excerpt: ''
tags:
  - SV
  - Pacio
  - Oxford Nanoporre
  - NGS methods
date: 2018-03-29
venue: 'Genome Research'
paperurl: 'https://genome.cshlp.org/content/early/2018/06/28/gr.231100.117.abstract'
citation: 'M. Nattestad, S. Goodwin, K. Ng, T. Baslan, FJ. Sedlazeck, P. Rescheneder, T. Garvin, H. Fang, J. Gurtowski, E. Hutton, E. Tseng, J. Chin, T. Beck, Y. Sundaravadanam, M. Kramer, E. Antoniou, J. McPherson, J. Hicks, WR. McCombie, MC Schatz. (2018). &quot;Complex rearrangements and oncogene amplifications revealed by long-read DNA and RNA sequencing of a breast cancer cell line.&quot; <i>Genome Research</i>.'
---

The SK-BR-3 cell line is one of the most important models for HER2+ breast cancers, which affect one in five breast cancer patients. SK-BR-3 is known to be highly rearranged although much of the variation is in complex and repetitive regions that may be underreported. Addressing this, we sequenced SK-BR-3 using long-read single molecule sequencing from Pacific Biosciences, and develop one of the most detailed maps of structural variations (SVs) in a cancer genome available with nearly 20,000 variants present, most of which were missed by short read sequencing. Surrounding the important ERBB2 oncogene (also known as HER2), we discover a complex sequence of nested duplications and translocations, suggesting a punctuated progression. Full-length transcriptome sequencing further revealed several novel gene fusions within the nested genomic variants. Combining long-read genome and transcriptome sequencing enables an in-depth analysis of how SVs disrupt the genome and sheds new light on the complex mechanisms involved in cancer genome evolution.

[Download paper here](https://www.nature.com/articles/s41592-018-0001-7)

